### PR TITLE
libkb: only use the legacy Unpack for loading from storage

### PR DESCRIPTION
**Pull request by maxtaco**

### CODE REVIEW

**Comment by maxtaco** [comment]
Seems to have passed CI

**Comment by zapu** [comment]
chain_link.go line 808
```
	// Beware that this is server-untrusted data at this point. We'll have to check it
	// before we can exit without error (see below).
```
this is no longer the case, right?

File: go/libkb/chain_link.go [hunk 1]
```diff
@@ -924,41 +924,39 @@ func (c *ChainLink) Unpack(m MetaContext, trusted bool, selfUID keybase1.UID, pa
...
-		}
+	}
+	if b, err := jsonparserw.GetBoolean(packed, "hash_verified"); err == nil && b {
+		c.hashVerified = true
+	}
+	if b, err := jsonparserw.GetBoolean(packed, "chain_verified"); err == nil && b {
+		c.chainVerified = true
+	}
+	if b, err := jsonparserw.GetBoolean(packed, "payload_verified"); err == nil && b {
+		c.payloadVerified = true
```
Comment (zapu) [inline go/libkb/chain_link.go]: are we saving to disk with any of the above being `false`? i think it's fine to leave this, just trying to make sure I understand

Comment (maxtaco) [reply go/libkb/chain_link.go]: right,  it's always `true`, was making minimal changes. thx!

**Pull request merged**